components/FlipTracking/FlipTracking.tsx

```diff
@@ -190,14 +191,59 @@ export function FlipTracking(props: Props) {
+                                    content={
+                                        <span>
+                                            <span className={styles.label}></span> {moment(trackedFlip.sellTime).fromNow()}
+                                        </span>
+                                    }
+                                    tooltipContent={<span>{trackedFlip.sellTime.toLocaleDateString() + ' ' + trackedFlip.sellTime.toLocaleTimeString()}</span>}
+                                />
+                            </span>
                         </p>
-                        <p style={{ marginTop: '10px' }}>Sell: {trackedFlip.sellTime.toLocaleDateString() + ' ' + trackedFlip.sellTime.toLocaleTimeString()}</p>
+                        <p
+                            style={{ cursor: 'pointer' }}
+                            onClick={() => {
+                                let flips = [...trackedFlips]
+                                trackedFlip.showPropertyChanges = !trackedFlip.showPropertyChanges
+                                setTrackedFlips(flips)
+                            }}
+                        >
+                            Profit changes: {trackedFlip.showPropertyChanges ? <ArrowDownIcon /> : <ArrowRightIcon />}
+                        </p>
+                        {trackedFlip.showPropertyChanges ? (
+                            <div style={{ display: 'flex', justifyContent: 'space-between' }}>
+                                <Table>
+                                    <tbody>
+                                        {trackedFlip.propertyChanges.map(change => (
+                                            <tr>
+                                                <td>{change.description}</td>
+                                                <td>
+                                                    {' '}
+                                                    {change.effect > 0 ? (
+                                                        <span style={{ color: 'lime', whiteSpace: 'nowrap', marginLeft: '5px' }}>
+                                                            +{numberWithThousandsSeperators(change.effect)} Coins
+                                                        </span>
+                                                    ) : (
+                                                        <span style={{ color: 'red', whiteSpace: 'nowrap', marginLeft: '5px' }}>
+                                                            {numberWithThousandsSeperators(change.effect)} Coins
+                                                        </span>
+                                                    )}
+                                                </td>
+                                            </tr>
+                                        ))}
+                                    </tbody>
+                                </Table>
+                            </div>
```

Comment:
looks like I hoped for except that there is now never more than one property 
![image](https://user-images.githubusercontent.com/16632490/193195787-6e56198a-d958-4729-b2dd-819400f6450f.png)
while the api says there is 
![image](https://user-images.githubusercontent.com/16632490/193195850-ea146f8f-c172-4bef-a117-8c7ed4ede841.png)
also, please use the profit returned from the api as your calculation seems to be wrong

Reply:
@Ekwav Could you please provide an example?
In my case it shows all the changes: 
![image](https://user-images.githubusercontent.com/58751503/193288702-27f75065-4788-407a-bb84-d67cb7524675.png)

Reply:
And I do use the profit property for that...
![image](https://user-images.githubusercontent.com/58751503/193289034-b5229613-76b0-4de2-8404-35f422bc6c10.png)

Reply:
The issue doesn't appear anymore. 
The auction in the screenshot(s) was http://localhost:3000/player/99ab148c61b146ed9b5d07df7b46984c/flips/9fc596827144

Reply:
The issue doesn't appear anymore. 
The auction in the screenshot(s) was http://localhost:3000/player/99ab148c61b146ed9b5d07df7b46984c/flips/9fc596827144